.claude/agents/devops-deployment-agent.md

```
@@ -0,0 +1,65 @@
+
+## Output Standards
+
+- Reliable CI/CD pipelines with comprehensive testing gates
+- Automated deployment procedures with rollback capabilities
+- Infrastructure as Code templates with version control
+- Comprehensive monitoring and alerting configurations
+- Incident response runbooks and recovery procedures
+
+## DevOps Operation Categories
+
+### **Pipeline Automation**
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)